codec_specs.md

```diff
@@ -1,21 +1,59 @@
+- CodecPrivate in the TrackEntry
+- CodecState in the BlockGroup
 
-*   CodecPrivate in the TrackEntry
-*   CodecState in the BlockGroup
-
-Each of these elements contain the same kind of data. And these data depend on the codec used.
+Each of these Elements contain the same kind of data. And these data depend on the codec used.
 
 Important Note:
 
-The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (matroska-devel at lists.matroska.org).
+The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (cellar at ietf.org).
+
+## Defining Matroska Codec Support
+
+Support for a codec is defined in Matroska with the following values.
+
+### Codec ID
+
+Each codec supported for storage in Matroska MUST have a unique Codec ID. Each Codec ID MUST be prefixed with a two character string according to the associated type of the codec as listed in the following table.
+
+Codec Type | Codec ID Prefix
+-----------|----------------
+Video      | V_
+Audio      | A_
+Subtitle   | S_
+Button     | B_
+
+Each Codec ID MUST then use a `Major Codec ID` and may optionally use a `Minor Codec ID` and `Micro Codec ID` with a forward-slash (`/`) delimiter between each component.
+
+For example a Codec ID MAY be constructed of the following components:
+
+Codec ID Prefix | Major Codec ID | Minor Codec ID | Micro Codec ID | Codec ID
+V_              | V_MPEG4        | ISO            | ASP            | V_MPEG4/ISO/ASP
+V_              | MPEG1          |                |                | V_MPEG1
+
+`Major Codec ID`, `Minor Codec ID`, and `Micro Codec ID` MUST only be composed of capital letters (A-Z) and numbers (0-9).
+
+### Codec Name
+
+Each codec supported for storage in Matroska MUST have a Codec Name. The Codec Name provides a readable label for the codec.
+
+### Description
+
+An optional description for the codec.
+
+### Initialisation
+
+Each codec supported for storage in Matroska MUST have a defined Initialisation. If the codec does not require any form of Initialisation then `none` SHOULD be used to define the Initialisation. The Initialisation MUST document in detail how data MUST be stored within the CodecPrivate and CodecState Elements in order to initialize the codec.
```

Comment:
«The Initialisation MUST document in detail how data MUST be stored» looks very weird to me. I suggest to replace the second MUST. – Yet this may be only my weird taste.